packages/styling/src/styles/theme.ts

```diff
@@ -144,20 +144,24 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
 // We'll use these as fallbacks for semantic slots that the passed in theme did not define.
 function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean, depComments: boolean): ISemanticColors {
   let toReturn: ISemanticColors = {
+    actionLink: p.neutralPrimary,
```

Comment:
put these down with the link slots (in this file and elsewhere)

Reply:
Moved actionlink slots to be grouped with link slots. Thanks!